resources/vue/src/components/site/trips/TripFilter.vue

```diff
@@ -0,0 +1,199 @@
+                            <input type="number" class="form-control" v-model="maxPrice" placeholder="Max Price" aria-label="Max Price">
+                        </div>
+                    </div>
+
+                </div>
+
+
+                <div class="col-5">
+
+                    <h4 class="mt-3">People</h4>
+                    <input-radio :options="peopleQuantities" :model-value="people" @update:model-value="people = $event" name="people"></input-radio>
+
+                    <h4 class="mt-3">Meal</h4>
+                    <input-radio :options="mealOptions" :model-value="meal" @update:model-value="meal = $event" name="meal"></input-radio>
+
+
+                    <h4>Date in</h4>
+                    <div class="row g-3 mb-3">
+                        <div class="col-auto">
+                            <input type="date" class="form-control" v-model="minDateIn" placeholder="Min date in" aria-label="Min date in">
+                        </div>
+                        <div class="col-auto">
+                            <input type="date" class="form-control" v-model="maxDateIn" placeholder="Max date in" aria-label="Max date in">
+                        </div>
+                    </div>
+
+                    <h4>Date out</h4>
+                    <div class="row g-3">
+                        <div class="col-auto">
+                            <input type="date" class="form-control" v-model="minDateOut" placeholder="Min date out" aria-label="Min date out">
+                        </div>
+                        <div class="col-auto">
+                            <input type="date" class="form-control" v-model="maxDateOut" placeholder="Max date out" aria-label="Max date out">
+                        </div>
+                    </div>
+
+                </div>
+            </div>
+
+            <div class="d-grid gap-2 d-md-flex justify-content-md-end mt-2" role="group" aria-label="Basic example">
+                <button type="submit" class="btn btn-primary">Submit</button>
+                <button type="reset" class="btn btn-danger">Reset</button>
+            </div>
+
+        </form>
+    </div>
+</template>
+
+<script>
+
+import InputSelect from "./InputSelect";
+import InputRadio from "./InputRadio";
+import {ref} from "vue";
+
+export default {
+    name: "trip-filter",
+
+    setup () {
+        const apiUrl = process.env.VUE_APP_API_ROOT_PATH + '/api',
+            tagsUrl = apiUrl + '/tags',
+            discountsUrl = apiUrl + '/discounts';
+        const tagOptions = ref([]);
+        const discountValues = ref([]);
+        const hotel = ref('');
+        const tag = ref('');
+        const discount = ref('');
+        const minPrice = ref('');
+        const maxPrice = ref('');
+        const people = ref('');
+        const meal = ref('');
+        const minDateIn = ref('');
+        const minDateOut = ref('');
+        const maxDateIn = ref('');
+        const maxDateOut = ref('');
```

Comment:
This can be simplified as a [reactive](https://v3.vuejs.org/api/basic-reactivity.html#reactive) object if you want to.